### PR TITLE
Added [p] whitelist show (and clear) & [p] blacklist show (and clear)

### PR DESCRIPTION
**Pull request by ddday**
Had requests to make handling easier soo.. as the title suggests i've added [p] whitelist show and clear + [p] blacklist show and clear
#202

### CODE REVIEW

File: cogs/mod.py [hunk 1]
```diff
@@ -1,540 +1,628 @@
-import discord
```
Comment (ddday) [inline cogs/mod.py:1]: If you're here just click the view button... I have no idea what happened here.

**Comment by irdumbs** [comment]
what's with the diff?
got carriage returns in there or something?

looks fine.
please PR to cog changes though

File: cogs/mod.py [hunk 2]
```diff
@@ -1,540 +1,628 @@
...
+    @blacklist.command(name="clear", pass_context=True)
+    async def _blacklist_clear(self, ctx):
+        """Resets the blacklist to a blank state"""
+        blklist = fileIO("data/mod/blacklist.json", "load")
+        await self.bot.say("Are you sure you want to clear the blacklist? Everyone who was previously blacklisted be able to use me that way. (yes/no)")
+        response = await self.bot.wait_for_message(author=ctx.message.author)
+        response = response.content.lower().strip()
+        if response == "yes" or response == "y":
+            for i in blklist:
+                self.blacklist_list.remove(i)
```
Comment (irdumbs) [inline cogs/mod.py:237]: could just do `self.blacklist_list = {}` instead of removing each one individually

**Comment by ddday** [comment]
@irdumbs Do I make a new PR? Because I am unable to edit that.

**Comment by tekulvw** [comment]
@ddday yes please do make a new pr

**Pull request closed without merging**